lib/route_format.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
 
 module RouteFormat
+  USERNAME_REGEXP = '[%\w.\-]+?'
```

Comment:
We might want to add a comment saying this regular expression is being used in JS as well (thus preventing us from using Ruby-specifics classes)